src/R8R.sol

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import {ReentrancyGuard} from "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import {IERC20} from "@openzeppelin/contracts/interfaces/IERC20.sol";
+
+contract R8R is Ownable, ReentrancyGuard {
+    // =========================
+    // === STORAGE VARIABLES ===
+    // =========================
+
+    address public robot;
```

Comment:
what is the robot variable? Maybe change name or provide comment for what this variable will do